roofit/roofitmore/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+############################################################################
+# CMakeLists.txt file for building ROOT roofitcore package
+# @author Pere Mato, CERN
+############################################################################
+
+if(mathmore OR ROOT_mathmore_FOUND)
+  set(ROOT_MATHMORE_LIBRARY MathMore)
```

Comment:
How are the GSL symbols resolved if `mathmore` is off? Shouldn't all of `roofitmore` only be built if `mathmore` is on?